R/skateboard_Laura.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: Skateboard
+output: github_document
```

Comment:
Awesome!
Can you add this?
author: Laura

Reply:
where should I add it ?

Reply:
Thanks